### PR TITLE
Hide DAOs from search

### PR DESCRIPTION
**Pull request by NoahSaso**
This hides some duplicate SubDAOs from search:

- juno1hgu46yeryyhcmjfr30h0607m4a48sd9penvt4sw85nftn5l3lgaqqlc99h
- juno1g325q3ddaxk943s52z9ejac4envfje98y0pnfw7lpnmxxdwdyw2sum2jgg
- juno12djpzq4q80qeywkvh60ym9ngpx2wee4x00uuyaxuqxh3x4skmumsnvth64
- juno1fydtvrjjja6dqpa88306n36u7lwjcz7mp263dzxfrfv4ckasaddqtj0laf
- juno14pugx8tx7zjfmcdknq7dnp3tkll6sq6jr7a0aemnfyhprvv747lqj7gunh
- juno16yfrarxm2u2865t8r4kkw6fzxfyp5pp9hfquqyk2sfqs4uuvreesusgpd7
- juno1fxa0jz3avrjjm6p5vsgrzxrytzfkzxhhpuuf03s53kmxk3nct96q0pjj6t
- juno12ra0ehc2ax93tnhyeqvtjzktnzlh2mnv5raqjeq5vrcfvyfff8ssk5uuce

### CODE REVIEW

**Review by elsehow — APPROVED** [review]
Let's spend 15 minutes talking about our near/mid/long-term search strategy.

File: packages/state/daos_hidden_from_search.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  "juno1hgu46yeryyhcmjfr30h0607m4a48sd9penvt4sw85nftn5l3lgaqqlc99h",
```
Comment (JakeHartnell) [inline packages/state/daos_hidden_from_search.json:2]:
```suggestion
// Duplicate Juno SubDAOs from testing / etc.
  "juno1hgu46yeryyhcmjfr30h0607m4a48sd9penvt4sw85nftn5l3lgaqqlc99h",
```

Like commenting a little rational for why DAOs are hidden. Potentially useful later if we ever have to revise the list or justify why certain DAOs are included here.

**Review by JakeHartnell — APPROVED** [review]
LGTM

**Comment by NoahSaso** [comment]
Closing due to #1049 which offers a better solution. Curating search will always be an issue, because duplicates are made all the time. We should deprioritize inactive DAOs instead of manually excluding individual DAOs from search.

**Pull request closed without merging**